src/ontology/mondo-edit.obo

```diff
@@ -440074,8 +440092,9 @@ xref: MESH:C536694 {source="UMLS:C1279186", source="MONDO:equivalentTo"}
 xref: OMIM:143100 {source="GARD:0005557", source="MONDO:relatedTo"}
 xref: SCTID:182747006 {source="UMLS:C1279186", source="MONDO:equivalentTo"}
 xref: UMLS:C1279186 {source="GARD:0005557", source="MONDO:equivalentTo"}
-is_a: MONDO:0000995 ! familial periodic paralysis
+is_a: MONDO:0007739 ! Huntington disease
```

Comment:
@sabrinatoro when there are new superclass assertions, could you add Peter's ORCID and also add the term tracker item to the ticket?

thank you!

Reply:
done